rustls-test/src/lib.rs

```diff
@@ -1715,6 +1715,40 @@ where
     }
 }
 
+/// Check `reader` has available exactly `bytes`
```

Comment:
Nit: this stuff doesn't seem to belong in this commit ("Address clippy::use_self in OtherSession")?